src/connectedComponents/ToolbarRow.js

```diff
@@ -96,6 +97,32 @@ class ToolbarRow extends Component {
   }
 
   render() {
+    const { t } = this.props;
+
+    const leftSidebarToggle = [
+      {
+        value: 'studies',
+        icon: 'th-large',
+        bottomLabel: t('Series'),
+      },
+    ];
+
+    const rightSidebarToggle = [
+      {
+        value: 'measurements',
+        icon: 'list',
+        bottomLabel: t('Measurements'),
+      },
+    ];
+
+    const leftSidebarValue = this.props.leftSidebarOpen
+      ? leftSidebarToggle[0].value
+      : null;
+
+    const rightSidebarValue = this.props.rightSidebarOpen
```

Comment:
I don't see where these variables are being used at all?

Reply:
It was after I merge with the latest master changes. I reverted already